README.md

```diff
@@ -50,112 +49,158 @@ This allows our [`dojo cli`](https://github.com/dojo/cli) build tooling to make
 
 - All widgets are designed to be accessible. If custom ARIA semantics are required, widgets have an `aria` property that may be passed an object with custom `aria-*` attributes.
 
-- All widgets are fully themeable. Example themes are available in the [@dojo/themes](https://github.com/dojo/themes) repository.
+- All widgets are fully themeable. Example themes are available in the [/theme](https://github.com/dojo/widgets/tree/master/src/theme) folder.
 
 - All widgets support internationalization (`i18n`)
 
-## Widgets
+## Examples
 
-Live examples of current widgets are available in the [widget showcase](https://dojo.github.io/examples/widget-showcase/).
+Live examples of current widgets are available at [widgets.dojo.io](https://widgets.dojo.io ).
```

Comment:
Now that we have Parade, I think we should expand this to `Documentation` and point out that we have examples, API docs, theme docs, etc all available here. I mean, it is a list of widgets, but it's so much more.